src/elements/element.line.js

```diff
@@ -21,10 +21,11 @@ module.exports = function(Chart) {
 		lineToNextPoint: function(previousPoint, point, nextPoint, skipHandler, previousSkipHandler) {
 			var me = this;
 			var ctx = me._chart.ctx;
+			var spanGaps = me._model ? me._model.spanGaps : false;
```

Comment:
`_view` is used for drawing. in this case it technically doesn't matter since it's a boolean, but it should use `_view` for consistency